odoo/addons/base/models/ir_ui_view.py

```diff
@@ -407,13 +407,14 @@ def _check_xml(self):
                     view_docs = view_docs[0]
                 for view_arch in view_docs:
                     check = valid_view(view_arch, env=self.env, model=view.model)
-                    view_name = ('%s (%s)' % (view.name, view.xml_id)) if view.xml_id else view.name
                     if not check:
+                        view_name = ('%s (%s)' % (view.name, view.xml_id)) if view.xml_id else view.name
                         raise ValidationError(_(
                             'Invalid view %(name)s definition in %(file)s',
                             name=view_name, file=view.arch_fs
                         ))
                     if check == "Warning":
+                        view_name = ('%s (%s)' % (view.name, view.xml_id)) if view.xml_id else view.name
```

Comment:
Alternatively: just gate the entire thing with 

    if check is True:
        continue

to avoid duplicating the formatting bit (and call to xml_id)?

Reply:
The message is translated in the error but left in english in the logs. They are similar but but exactly the sames, I argue it is ok to leave it this way